.github/workflows/ubuntu-lint.yml

```diff
@@ -21,7 +21,7 @@ jobs:
         run: rake setup
       - name: Run Lint
         run: |
-          gem install rubocop -v "~>0.74.0"
+          gem install rubocop -v "~>0.80.1"
```

Comment:
Can you verify whether we actually need this? I'd say we don't because `rake setup` should take care of it.